EIPS/eip-7522.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
```suggestion
---
eip: 7503
```
Assigning next sequential EIP/ERC number.  
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number: https://github.com/ethereum/EIPs/issues/6990#issuecomment-1648535814

Please also update the filename

Reply:
Done, thanks